tests/core/test_macros.py

```diff
@@ -76,6 +78,17 @@ def test_macro_var(macro_evaluator):
         macro_evaluator.locals = {"x": k}
         assert macro_evaluator.transform(expression).sql() == v
 
+    # Check that macro vars are rendered as-is if they don't map to something in locals
+    e = parse_one("select * from @staged_table", dialect="snowflake")
```

Comment:
Can you please also add a test for snowflake where the macro syntax is used in eg. projection

Reply:
Sure, good point